### DISCUSSION TITLE
How to edit minPxPerSec property?

### DISCUSSION THREAD

**Discussion by ngocducdim** [katspaugh/wavesurfer.js#2698] [Q&A] [answered]
Hi can I ask how can I modify `minPxPerSec ` property? Thanks

**Comment by katspaugh** [comment, marked as answer]
It's an option when you create a wavesurfer.js instance. `scrollParent` needs to be set to true for it to work.
See https://codesandbox.io/s/aged-violet-kvbqut?file=/index.html
